services/webnn/ort/context_impl_ort.cc

```diff
@@ -245,6 +247,14 @@ ContextProperties ContextImplOrt::GetContextProperties() {
       OperandDataType::kFloat16, OperandDataType::kFloat32,
       OperandDataType::kInt32};
 
+  bool use_ov_gpu =
+      base::FeatureList::IsEnabled(mojom::features::kWebNNOrtOpenVino) &&
+      device_type == mojom::CreateContextOptions::Device::kGpu;
+  // OV EP GPU does not support integer data type for sign operation.
```

Comment:
Could you add a pointer of OV document or source code for this limit?

Reply:
Done. PTAL, thanks!